robotoff/insights/ocr/packager_code.py

```diff
@@ -29,13 +28,24 @@ def process_fr_emb_match(match) -> str:
     return "EMB {}{}".format(city_code, company_code).upper()
 
 
+def process_fsc_match(match) -> str:
+    fsc_code = match.group(1)
+    return "FSC-{}".format(fsc_code)
+
+
 PACKAGER_CODE: Dict[str, OCRRegex] = {
     "fr_emb": OCRRegex(
         re.compile(r"emb ?(\d ?\d ?\d ?\d ?\d) ?([a-z])?(?![a-z0-9])"),
         field=OCRField.text_annotations,
         lowercase=True,
         processing_func=process_fr_emb_match,
     ),
+    "fsc": OCRRegex(
+        re.compile(r"FSC.? (C\d{6})"),
+        field=OCRField.text_annotations,
+        lowercase=False,
```

Comment:
any reason why not lowercasing? The pattern seems specific enough to me to allow lowercasing.

Reply:
@raphael0202 @teolemon 
I updated the branch with the modifications